test/generate-build-config

```diff
@@ -35,7 +32,7 @@ def generate_manifests(outputdir):
     Generate all manifest using the default config map and return a dictionary mapping each manifest file to the
     manifest data and its ID.
     """
-    arches_arg = ",".join(ARCHITECTURES)
+    arches_arg = os.uname().machine
```

Comment:
Not sure if a person would run this script locally in some valid case, but doing this will make it impossible to generate the config for other architecture locally.

Reply:
Sure. I had the impression that this script is not intended to be run by a user, just wanted to double-check... A followup sounds OK to me, but not adding the option if not needed is also OK from my PoV.